setup.py

```diff
@@ -44,32 +41,39 @@ def package_data(pkg, root_list):
 
 BLOCKS = [
     'mentoring = mentoring:MentoringBlock',
-    'mentoring-dataexport = mentoring:MentoringDataExportBlock',
-]
 
-BLOCKS_CHILDREN = [
     'mentoring-table = mentoring:MentoringTableBlock',
-    'column = mentoring:MentoringTableColumnBlock',
-    'header = mentoring:MentoringTableColumnHeaderBlock',
+    'mentoring-column = mentoring:MentoringTableColumn',
     'answer = mentoring:AnswerBlock',
-    'quizz = mentoring:QuizzBlock',
-    'message = mentoring:MentoringMessageBlock',
-    'tip = mentoring:QuizzTipBlock',
-    'choice = mentoring:QuizzChoiceBlock',
-    'html = mentoring:HTMLBlock',
+    'answer-recap = mentoring:AnswerRecapBlock',
+    'mcq = mentoring:MCQBlock',
+    'rating = mentoring:RatingBlock',
+    'mrq = mentoring:MRQBlock',
+    'mentoring-message = mentoring:MentoringMessageBlock',
+    'tip = mentoring:TipBlock',
```

Comment:
I can understand why you want to name these blocks this way, but this greatly increases the chances of colliding with other blocks that want to be called "tip" (for example).  Ideally, we'd have a way for you to use xml namespaces to deal with this, but we don't.